loader.py

```diff
@@ -218,6 +218,9 @@ def main(args):
                 if not confirm_deletion('Wipe out entire Neo4j database before loading?'):
                     sys.exit(1)
 
+            if config.loading_mode == DELETE_MODE:
+                config.cheat_mode = True
```

Comment:
Why we need to set cheat_mode here?

Reply:
Because of the "Disable all other validations. For example, required properties, relationships, etc." requirement